sections/video.liquid

```diff
@@ -0,0 +1,141 @@
+{{ 'video-section.css' | asset_url | stylesheet_tag }}
+{{ 'component-deferred-media.css' | asset_url | stylesheet_tag }}
+
+<div class="video-section{% unless section.settings.full_width %} page-width{% endunless %}">
+  <h2 class="video-section__title title{% if section.settings.full_width %} page-width{% endif %}">{{ section.settings.heading }}</h2>
+  <div class="video-section__wrapper{% if section.settings.image_padding %} video-section__media--padding{% endif %}">
+    <noscript>
+      <div class="video-section__media"
+        {% if section.settings.cover_image != blank %}
+          style="padding-bottom: {{ 1 | divided_by: section.settings.cover_image.aspect_ratio | times: 100 }}%;"
+        {% else %}
+          style="padding-bottom: 56.25%;"
+        {% endif %}
```

Comment:
Small nitpick: can we reduce the number of lines and merge into 1 line? For example

```suggestion
          style="padding-bottom: {% if section.settings.cover_image != blank %}{{ 1 | divided_by: section.settings.cover_image.aspect_ratio | times: 100 }}{% else %} padding-bottom: 56.25{% endif %}%;"
```

Or we can move the `padding-bottom: 56.25%` into `.video-section__media` class as default value. When the `{% if section.settings.cover_image != blank %}` condition is satisfied, it will overwrite the default `56.25%` as it's an inline style, which has higher priority.

Reply:
Great idea!